StyleGuide.md

```diff
@@ -970,6 +973,73 @@ Use the following HTML and Liquid to include an image in a Markdown page:
 
 Example: [Decommission Nodes](https://www.cockroachlabs.com/docs/stable/remove-nodes.html#step-1-check-the-node-before-decommissioning)
 
+<a name="include-files"></a>
+
+### Include files
+
+Sometimes content needs to be duplicated across two or more pages in the documentation. For example, there may be several pages that need the same cluster setup, but describe how to use different features. Or a very specific [note](#notes) or [warning](#warnings) needs to be added to several different pages.
+
+In these situations, you will need to use something we call an _include file_. An include file is a separate Markdown file (stored in `_includes/{version}/some/shared-file.md`) where you will write content to be shared across multiple pages.
```

Comment:
There's also `common` (non-version specific includes) and `cockroachcloud`.

Reply:
I changed this from `{version}` to `_includes/some/shared-file.md` and then added the `common` folder to the list of folders discussed below in the 'Basic' section